src/mappings/util/metadata.ts

```diff
@@ -21,7 +21,7 @@ export async function fetchMetadata(url: string, job: Queue.Job) {
         },
         withCredentials: false,
         timeout: 15000,
-        maxRedirects: url.startsWith('https://platform.production.enjinusercontent.com/') ? 3 : 1,
+        maxRedirects: url.startsWith('http://platform.production.enjinusercontent.com/') ? 3 : 2,
```

Comment:
Why it is needed to change `https` to `http`?

Reply:
it was implemented wrong, if URL starts with http://, then it will require an extra redirect because it will first redirect to https://